server/routes/routers/users.ts

```diff
@@ -7,16 +7,18 @@ const prisma = new PrismaClient();
 
 // Authenticated route to verify a user is logged in
 users.get('/loggedIn', (req: any, res: any) => {
-  const user = req.user;
   if (req.isAuthenticated()) {
-    res.send({id: user.id});
+    const user = req.user;
+    res.json({ isLoggedIn: true, id: user.id });
   } else {
-    res.send({id: 0})
+    res.json({ isLoggedIn: false, id: 0 });

```

Comment:
What is the benefit of using`res.json` over `res.send`?

Reply:
From what I was reading it is best practice to use json over send since res.json is tailored for json responses. It also automatically sets the content-type header to 'application/json'